test/test_throughput_benchmark.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from common_utils import run_tests, TestCase
+from torch.jit import ThroughputBenchmark, BenchmarkConfig
+from torch.testing import assert_allclose
+
+from tempfile import NamedTemporaryFile
+
+class TwoLayerNet(torch.jit.ScriptModule):
+    def __init__(self, D_in, H, D_out):
+        super(TwoLayerNet, self).__init__()
+        self.linear1 = torch.nn.Linear(D_in, H)
+        self.linear2 = torch.nn.Linear(H, D_out)
+
+    @torch.jit.script_method
+    def forward(self, x):
+        h_relu = self.linear1(x).clamp(min=0)
+        y_pred = self.linear2(h_relu)
+        return y_pred
+
+class TestThroughputBenchmark(TestCase):
+    def test_linear_runs(self):
+        D_in = 10
+        H = 5
+        D_out = 15
+        B = 8
+        NUM_INPUTS = 2
+
+        module = TwoLayerNet(D_in, H, D_out)
+        with NamedTemporaryFile() as f:
+            torch.jit.save(module, f.name)
+
+        config = BenchmarkConfig()
```

Comment:
why not just make them arguments of ThroughputBenchmark function?

or even better - make them arguments of .benchmark() method

Reply:
Oh, named arguments, nice! :)

Reply:
@dzhulgakov , actually having a config struct should be useful going forward. One can supply million of various knobs. Function signature will become really bad on the c++ side if we keep pushing those as arguments to a function. 

For python we can have a wrapper which under the hood creates the config structure and passes it over to c++.